src/Common/ExecCommand.php

```diff
@@ -124,13 +124,24 @@ protected function useCallOnWindows($cmd)
         return $cmd;
     }
 
+    public function getCommand() {
+        return $this->process->getCommandLine();
+    }
+
     /**
      * @param string $command
      *
      * @return \Robo\Result
      */
     protected function executeCommand($command)
     {
-        return $this->execute($command);
+        $this->process = new Process($command);
+        $result_data = $this->execute();
```

Comment:
Seems better to have the `execute()` method take a Process as a parameter. Other places would then become `$this->execute($this->process);`, but here you could just pass a local Process object.